integration_tests/Cargo.toml

```diff
@@ -40,8 +41,10 @@ serde.workspace = true
 serde_json = "1.0.108"
 testcontainers.workspace = true
 tokio.workspace = true
-url = { version = "2.5.0", features = ["serde"] }
+trillium-rustls.workspace = true
```

Comment:
What's the rationale for having `trillium-rustls` be a workspace dependency when it's only used once? I've actually been grappling with "when to use workspace dependency vs. not" so was curious as to your thoughts--I don't feel strongly one way or another.

Reply:
Hmm, this might be rebase carnage because we did happen to use `trillium-rustls` in more than one crate at some point. I don't feel very strongly about this one way or the other, either. My only thought is that we will probably end up using `trillium-rustls` in another Janus crate at some point, and so we'll have saved ourselves the trouble of moving the dependency by eagerly putting it in the top-level `Cargo.toml`.